### ISSUE TITLE
Increase Contrast

### ISSUE THREAD

**Issue by hannahackermans** [smbjorklund/cellproject#62]
### Current State
There are several problems with the contrast. 

- The green (2.5:1) and grey text (1.8:1) do not have enough contrast with the background. 
- Moreover, links are only indicated by color even if you have keyboard focus or hover on them. 
- the search box has a very low contrast (1.4:1)

### Proposed changes to meet the success criteria

1. Increase contrast for all green and grey text and buttons (which have text in them) to 4.5:1 contrast. 
2. To fix the failure of only color is used to indicate a link, technique G183 states that "Using a contrast ratio of 3:1 with surrounding text and providing additional visual cues on hover for links or controls where color alone is used to identify" is also "sufficient" to meet the 1.4.1 Use of Color guideline. Please note that the 3:1 is between the linked text and the regular text color, but that both of these colors need a 4.5:1 contrast ration with the background.This is a webpage with working examples: [Links with a 3:1 contrast ratio with surrounding text](https://www.w3.org/WAI/WCAG21/working-examples/link-contrast/)
Please note that the list of colors on this page would pass if the background is white and the regular text is black. Our background is white, but our regular text is currently grey, so we would either need to find an alternative color or make the regular text darker.
4. increase the contrast on the search box to 3:1 contrast. 

NOTE: a related issue is the lack of keyboard focus indicator, see issue #63
